### DISCUSSION TITLE
Copilot Inline suggestions stopped working since today morning

### DISCUSSION THREAD

**Discussion by acehand** [community/community#51972] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I have been using vscode insider with Copilot. Today morning, after the new update, v1.79.10634 , Copilot inline suggestion stopped working. I can still click on the `Copilot` extension to open the suggestion Tab and can see suggestions but no luck getting the tabbing to get inline suggestions to work. 

I have tried things like disabling all extensions except copilot, restarting , re-installing etc.

**Comment by SMelidoni** [comment]
Hi there 👋🏽

If Copilot inline suggestions stopped working after an update to VS Code Insider, there are a few things you can try to troubleshoot the issue:

1. Make sure the Copilot extension is up to date. Check for updates in the Extensions view in VS Code Insider and ensure that the Copilot extension is at the latest version.

2. Try resetting your VS Code Insider settings to default. This can be done by going to the Command Palette (View > Command Palette or by pressing `Ctrl+Shift+P` on Windows or `Cmd+Shift+P` on Mac) and running the "Preferences: Open User Settings (JSON)" command. Once the JSON file opens, delete all the contents and save the file. This will reset your VS Code Insider settings to their default values.

3. Try clearing the VS Code Insider cache. You can do this by running the "Developer: Purge Cache" command from the Command Palette.

I hope this helps you resolve the issue with Copilot inline suggestions.

**Reply by acehand** [reply]
Have tried the first 2. Am not sure there is a `Purge Cache` command that I can find in command palette.  But I have tried to clear cache in general.  I have a feeling there is an issue either with VsCode Insider or CoPilot

**Reply by tomgreenhill** [reply]
Same here, no `Purge Cache` command for me either. Still not working...

**Comment by ductai199x** [comment, marked as answer]
Guys, the fix is coming. Don't mess around with your configs, they are not the problem.
https://github.com/microsoft/vscode/pull/179171

**Reply by War3Evo** [reply]
It's been 3 days, I've all the updates, no luck getting inline suggestions.

**Reply by acehand** [reply]
Hey @War3Evo  - Once they released the update, it started working for me. Are you sure plugin is enabled

**Reply by War3Evo** [reply]
For some reason it's finicky.  It isn't as responsive as it used to be.

I have to be fast coding on my own for about an hour before it started giving any response.  I'm not sure why, but it starts giving suggestions after I've been solo for an hour.  This happens everytime I get started on the morning.

I watch the log output (message ID headers, etc), I can see the service respond, but I'm guessing it's not getting any suggestions until I've been coding for quite some time on my own.

I'm not sure why this change?


I've got the latest nightly update and the latest VS code.

**Reply by acehand** [reply]
Actually I agree too. It has become very slow after the update. Anyone else face similar issues ?

**Comment by WeaponizedLego** [comment]
For anyone else falling over this. When I installed the insider version with github copilot chat, instructions said to uninstall the old release version. 

After a while the chat version had inline suggestion stop working, and I waited patiently for a update. 

Today I just installed the released version of copilot based on the inline suggestions and have it running alongside the chat version. Both versions seem to run fine along side each other, so until chat version get's inline suggestions again I'll use this solution to have both chat and inline copilot.

**Comment by ronaldroe** [comment]
For people who land here after me, I don't know how it happened, but the setting `github.copilot.enable` had been set with a wildcard to `false`. This tells copilot to never provide suggestions. I switched the wildcard to true, and it worked

**Reply by Mishkile** [reply]
I was totally lost when I noticed copilot was making suggestions but not pasting them to my editor.
it was the same for me, chaning to true worked.

**Comment by Entretoize** [comment]
I don't receive suggestion anymore, where can I find `github.copilot.enable` ?

**Reply by Entretoize** [reply]
I found it but it is true...
